LocalQueryTest.when_queryFromFollower_onStableCluster_thenReadLatestValue

```
23:32:18 [ERROR] when_queryFromFollower_onStableCluster_thenReadLatestValue(com.hazelcast.cp.internal.raft.impl.LocalQueryTest)  Time elapsed: 0.023 s  <<< FAILURE!
23:32:18 java.lang.AssertionError: 
23:32:18 
23:32:18 Expected: (is "value2" or is "value2")
23:32:18      but: was "value3"
23:32:18 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
23:32:18 	at org.junit.Assert.assertThat(Assert.java:956)
23:32:18 	at org.junit.Assert.assertThat(Assert.java:923)
23:32:18 	at com.hazelcast.cp.internal.raft.impl.LocalQueryTest.when_queryFromFollower_onStableCluster_thenReadLatestValue(LocalQueryTest.java:127)
23:32:18 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
23:32:18 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
23:32:18 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
23:32:18 	at java.lang.reflect.Method.invoke(Method.java:498)
23:32:18 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
23:32:18 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
23:32:18 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
23:32:18 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
23:32:18 	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
23:32:18 	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
23:32:18 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
23:32:18 	at java.lang.Thread.run(Thread.java:748)
```

--- a/hazelcast/src/test/java/com/hazelcast/cp/internal/raft/impl/LocalQueryTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/cp/internal/raft/impl/LocalQueryTest.java
@@ -34,13 +34,10 @@ import org.junit.runner.RunWith;
 import static com.hazelcast.cp.internal.raft.impl.RaftUtil.getCommitIndex;
 import static com.hazelcast.cp.internal.raft.impl.RaftUtil.getLeaderMember;
 import static com.hazelcast.cp.internal.raft.impl.testing.LocalRaftGroup.LocalRaftGroupBuilder.newGroup;
-import static org.hamcrest.Matchers.either;
-import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThat;
 
 @RunWith(HazelcastParallelClassRunner.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
@@ -119,12 +116,14 @@ public class LocalQueryTest extends HazelcastTestSupport {
             leader.replicate(new ApplyRaftRunnable("value" + i)).get();
         }
 
-        RaftNodeImpl[] followers = group.getNodesExcept(leader.getLocalMember());
-        Object result0 = followers[0].query(new QueryRaftRunnable(), QueryPolicy.ANY_LOCAL).get();
-        Object result1 = followers[1].query(new QueryRaftRunnable(), QueryPolicy.ANY_LOCAL).get();
-
         String latestValue = "value" + count;
-        assertThat(latestValue, either(is(result0)).or(is(result1)));
+        RaftNodeImpl[] followers = group.getNodesExcept(leader.getLocalMember());
+        assertTrueEventually(() -> {
+            for (RaftNodeImpl follower : followers) {
+                Object result = follower.query(new QueryRaftRunnable(), QueryPolicy.ANY_LOCAL).get();
+                assertEquals(latestValue, result);
+            }
+        });
     }
 
     @Test

